helm/fluent-logshipping-app/values.yaml

```diff
@@ -48,6 +54,7 @@ outputs:
   azure:
     logAnalytics:
       enabled: false
+      inputLogTypes: *inputTypes
```

Comment:
Should we not name it logTypes instead?

Reply:
I am OK with `logTypes`, Just in my head at the time I was thinking I am matching against the inputs